test/cmd/apply.sh

```diff
@@ -94,7 +94,7 @@ run_kubectl_apply_tests() {
   output_message=$(! kubectl get pods selector-test-pod-dont-apply 2>&1 "${kube_flags[@]:?}")
   kube::test::if_has_string "${output_message}" 'pods "selector-test-pod-dont-apply" not found'
   # cleanup
-  kubectl delete pods selector-test-pod
+  kubectl delete "${kube_flags[@]:?}" pods selector-test-pod
```

Comment:
global comment - I expected to set $KUBECONFIG pointing at a file containing credentials so we don't have to set the token in the invocation

Reply:
This is a good point, thanks for pointing it out! since its already midnight here, I would make this change tomorrow.